### PR TITLE
Don't rebase GeckoView or Android-Components version bumps PRs before landing

### PR DESCRIPTION
**Pull request by bhearsum**
@csadilek, @rvandermeulen and I talked about this a bit and thought it would probably be OK. @pocmo, what do you think?

### CODE REVIEW

File: .mergify.yml [hunk 1]
```diff
@@ -15,8 +15,11 @@ pull_request_rules:
         type: APPROVE
         message: MickeyMoz 💪
       merge:
-        method: rebase
-        strict: smart
+        # These PRs are just bumping versions, and very unlikely to break
+        # anything. It's not worth the delay of rebasing them
+        # (sometimes multiple times) before merging.
+        method: squash
+        strict: false
```
Comment (pocmo) [inline .mergify.yml:22]: For the bump PRs I guess we can try this. In this case I think we should do the same for the L10N Auto Merge below too.

File: .mergify.yml [hunk 2]
```diff
@@ -15,8 +15,11 @@ pull_request_rules:
         type: APPROVE
         message: MickeyMoz 💪
       merge:
-        method: rebase
-        strict: smart
+        # These PRs are just bumping versions, and very unlikely to break
+        # anything. It's not worth the delay of rebasing them
+        # (sometimes multiple times) before merging.
+        method: squash
```
Comment (pocmo) [inline .mergify.yml:21]: nit: Wondering why we switch to `squash` here? Couldn't this be `rebase` with `strict: false`? I guess it doesn't matter much for the "single commit" PRs that the bot currently opens.

But apart from that, I just checked the ([docs](https://docs.mergify.io/actions/merge/) and it looks like the merge options are deprecated and have now more fine-grained queue controls (although only for premium plans?).

**Review by pocmo — APPROVED** [review]

File: .mergify.yml (hunk 1 above)
Comment (bhearsum) [reply .mergify.yml:22]: Sure!

File: .mergify.yml (hunk 2 above)
Comment (bhearsum) [reply .mergify.yml:21]: Yeah, that's a good point. When I wrote this I thought that we couldn't rebase without being up-to-date, but that's silly in retrospect. I'll revert this part.

**Pull request merged**